docs/howtos/fhir-conversions.rst

```diff
@@ -0,0 +1,64 @@
+FHIR Conversions
+========================
+
+CLI
+===============
+The general pattern is from the CLI is:
+
+.. code-block:: bash
+
+    $ runoak -i INPUT-SPEC dump -O FORMAT -o OUTPUT-FILE
+
+The `fhirjson` format includes a --config-file / -c parameter for its additional parameters.
+
+Example:
+
+.. code-block:: bash
+
+    $ runoak --i go-nucleus.obo dump -o CodeSystem-go-nucleus.json -O fhirjson -c tests/input/fhir_config_example.json
+
+Config file
+===============
+
+Options:  
+
+code_system_id: _The code system ID to use for identification on the server uploaded to.
```

Comment:
I like how you added a how-to but I don't think you need to restate everything here. The role of the how-to is to provide additional explanation about why something is done.

See https://diataxis.fr/